extension/content.js

```diff
@@ -207,6 +207,58 @@ function indentInput(el, size = 4) {
 	el.selectionEnd = el.selectionStart = selectionStart + indentationText.length;
 }
 
+function lineUpInput(el, down) {
+	el.focus();
+	const value = el.value;
+	const selectionStart = el.selectionStart;
+	const selectionEnd = el.selectionEnd;
+	const lines = value.split(/(\r?\n)/);
```

Comment:
Oh. I forgot why I did this. You need to know the character length of the newline. Otherwise selectionStart / selectionEnd will be unreliable